Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -27,6 +27,30 @@ Before the testing library can be used, it must be added as a dependency of your
 Swift package or Xcode project. For more information on how to add it, see the
 [Getting Started](doc:TemporaryGettingStarted) guide.
 
+### Importing the testing library
+
+XCTest and the testing library are available from different modules. Instead of 
+importing the XCTest module, import the Testing module:
+
+@Row {
+  @Column {
+    ```swift
+    // Before
+    import XCTest
+    ```
+  }
+  @Column {
+    ```swift
+    // After
+    import Testing
+    ```
+  }
+}
+
+A single source file can contain tests written with XCTest as well as other 
```

Comment:
This could be re-phrased so it's more concise:

Using the two frameworks in a single file is supported. To do so, import both XCTest and Testing.

Reply:
swift-testing is a package though, not a framework. So that may be confusing.

Reply:
Replace framework with library. Since there are a lot of situations where those terms are used interchangeably in day-to-day work, that casualness slipped into my suggestion. Aside from that one word, I think that sentence is a better explanation.

Reply:
Should I change to:

```
Using the two libraries in a single file is supported. To do so, import both XCTest and Testing.
```

?

Reply:
I think what we've got here is fine as-is. @SeanROlszewski how strongly do you feel about it?

Reply:
Sounds good to me!

Reply:
Great! @Sajjon If you're ready to merge, let me know.

Reply:
@grynspan good to merge IMO.

What do I need to do? Update source branch with head? Rebase or merge main into source? 

Looks like there are no conflicts so GitHub ought to be able to merge it as is (I m on GitHub iOS app currently so not as "glanceable")

Reply:
I'll merge for you since you can't click the big green button yourself. :)